src/sidebar/util/time.js

```diff
@@ -195,48 +198,49 @@ function nextFuzzyUpdate(date, now) {
  * update frequency depends on the age of a timestamp.
  *
  * @param {String} date - An ISO 8601 date string timestamp to format.
- * @param {Date} callback - A callback function to call when the timestamp changes.
+ * @param {Function} callback - A callback function to call when the timestamp changes.
  * @return {Function} A function that cancels the automatic refresh.
  */
 function decayingInterval(date, callback) {
   let timer;
   const timeStamp = date ? new Date(date) : null;
-  const update = function() {
+
+  const update = () => {
     const fuzzyUpdate = nextFuzzyUpdate(timeStamp, new Date());
     if (fuzzyUpdate === null) {
       return;
     }
-    const nextUpdate = 1000 * fuzzyUpdate + 500;
-    timer = setTimeout(function() {
+    const nextUpdate = fuzzyUpdate + 500;
```

Comment:
The magic number `500` here...presume there's some reason we want to delay by half a second but it's not documented anywhere.

Reply:
Probably! I just pointed it out because it was undocumented.

Reply:
> The magic number 500 here...presume there's some reason we want to delay by half a second but it's not documented anywhere.

Kyle is correct, it's just a reasonably long delay before updating the timestamp.